sql-cloudant/src/main/scala/org/apache/bahir/cloudant/common/JsonStoreConfigManager.scala

```diff
@@ -35,6 +35,7 @@ object JsonStoreConfigManager {
   private val CLOUDANT_PASSWORD_CONFIG = "cloudant.password"
   private val CLOUDANT_PROTOCOL_CONFIG = "cloudant.protocol"
   private val CLOUDANT_API_ENDPOINT = "cloudant.endpoint"
+  private val CLOUDANT_STREAMING_BATCH_INTERVAL = "cloudant.batchInterval"
   private val STORAGE_LEVEL_FOR_CHANGES_INDEX = "cloudant.storageLevel"
```

Comment:
This still says `cloudant.storageLevel` but the documentation was changed to `storageLevel`, shouldn't they be the same?

Reply:
Yes, they should be the same.  I'll revert the documentation back to `cloudant.storageLevel`.

Reply:
Fixed in f0ae901.